css/style.css

```diff
@@ -15,103 +13,192 @@ body {
-    color: #838994;
-}
-
-.bold {
-    font-weight: bold;
-}
-
-.bold hover {
-    background-color: bisque;
-}
-
-.button {
-    width: 500px;
-    height: 200px;
-    padding: 40px 60px;
-}
-
-
-/* Introduction styling */
-
-.introduction {
-    background-image: url(../img/first-background.jpg);
-    height: 45.938rem;
-}
-
-
-/* Boxes styling */
-
-.features {
-    display: flex;
-    justify-content: center;
-}
-
-.caption {
-    position: relative;
-    top: 150px;
-    right: 160px;
-    width: 200px;
-    text-align: center;
-}
-
-.links li,
-.features li {
-    list-style-type: none;
-}
-
-footer {
-    border-top: 1px solid lightgrey;
-    margin-left: 120px;
-    margin-right: 120px;
-}
\ No newline at end of file
+    
+    * {
+        box-sizing: border-box;
+        font-size: 100%;
+        margin: 0;
+        border: 0;
+        padding: 0;
+        font-family: inherit;
+    }
+    
+    body,
+    html {
+        width: 100%;
+        height: 100%;
+    }
+    
+    body {
+        line-height: 1.2;
+    }
+    /* Header section styling */
+    
+    .upper-section .logo {
+        height: 3em;
+        position: fixed;
+        top: 1em;
+        left: 20em;
+        z-index: 1;
+    }
+    
+    .navigation {
+        width: 100%;
+        background-color: #FFF;
+        position: fixed;
+        display: flex;
+        justify-content: flex-end;
+        z-index: 1;
+    }
+    
+    .navigation .links {
+        margin-left: 18em;
+        margin-right: 12em;
+    }
+    
+    .navigation .links li {
+        display: inline-block;
+        color: #838994;
+        margin: 1.7em 0.9em;
+    }
+    
+    .navigation .links li:hover {
+        color: #333;
+        cursor: pointer;
+    }
+    
+    .navigation .links .bold {
+        font-weight: 500;
+        color: #4c5058;
+    }
+    /* Introduction styling */
+    
+    .button {
+        width: 10em;
+        height: 3.3em;
+        padding: 16px 20px;
+        background-color: #EF6327;
+        border-radius: 0.3em;
+    }
+    
+    .button:hover {
+        background-color: #E05616;
+    }
+    
+    .button p {
+        color: #FFF;
+        font-size: 1.1em;
+        text-align: center;
+    }
+    
+    .introduction {
+        background-image: url(../img/first-background.jpg);
+        background-size: cover;
+        background-position: center bottom;
+        height: 50em;
+        overflow: auto;
+    }
+    
+    .description {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+    
+    h1,
+    h3 {
+        font-weight: 300;
+        color: #FFF;
+        text-align: center;
+        line-height: 0.4em;
+    }
+    
+    h1 {
+        font-size: 3em;
+        margin-top: 7em;
+    }
+    
+    h3 {
+        font-size: 1.7em;
+        margin-top: 0.2em;
+        margin-bottom: 1.2em;
+    }
+    /* Boxes styling */
+    
+    .boxes {
+        position: relative;
+        left: 2em;
```

Comment:
This rule is making that section mis-align on mobile. Remove it, and add flexbox rules to each `li` to center the icon and text inside each list item.